src/SWP/Bundle/AnalyticsBundle/Controller/StatisticsController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Analytics Bundle.
+ *
+ * Copyright 2017 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2017 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\AnalyticsBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/statistics")
+ */
+class StatisticsController extends Controller
+{
+    /**
+     * @Route("/receive")
+     *
+     * @Method("GET|POST")
+     *
+     * Receive events
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function receiveAction()
+    {
+        return new Response('received');
```

Comment:
This is missing the functionality, why do you push this controller at all?

Reply:
i will improve it later this week, don't want to remove it now.